src/relay/mutationUtils.tsx

```diff
@@ -0,0 +1,73 @@
+import { ConnectionHandler, RecordProxy, RecordSourceProxy } from 'relay-runtime';
+
+type ConnectionUpdater = {
+  store: RecordSourceProxy;
+  parentId: string;
+  connectionName: string;
+  edge: RecordProxy;
+  before?: boolean;
+};
+
+export function connectionUpdater({ store, parentId, connectionName, edge, before = false }: ConnectionUpdater) {
```

Comment:
you do not need these helpers since now has https://relay.dev/docs/guided-tour/list-data/updating-connections/